### ISSUE TITLE
Windows: netclient causes high CPU usage on wireguard: "Unable to ingest and encrypt"

### ISSUE THREAD

**Issue by janost** [gravitl/netmaker#559]
Netclient causes the wireguard manager process to spam errors in its log and constantly hammer the CPU.
I was able reproduce the issue on 2 separate Windows 10 computers.
The issue seems to happen randomly, sometimes it doesn't happen for several hours, sometimes it happens instantly after the computer boots up.
Netclient version: v0.9.2

![wg-win](https://user-images.githubusercontent.com/11500057/147012175-5a58e049-66b6-4264-984c-f3711a75d79d.png)
![wg-task](https://user-images.githubusercontent.com/11500057/147012223-e2505fbb-a66c-4aa6-b6f8-8af995b2fdaf.png)
![wg-version](https://user-images.githubusercontent.com/11500057/147012227-3c1063ef-9b6a-41a1-ad3b-392e1301300a.png)

**Comment by QZAiXH** [comment]
I encountered the same situation as you, did you solve it?

**Comment by janost** [comment]
No, I did not. There was a wireguard app update about 2 days ago to version 0.5.3 and I believe I haven't encountered the issue since, but I'm not 100% sure.

**Comment by Bpazy** [comment]
I have the same situation with Wireguard version 0.5.3.

In my test, I found that occurred when `netclient` retrieves new configuration and triggers `wireguard` reconnecting.

**Comment by afeiszli** [comment]
Adding to triage. As a workaround for users, as @Bpazy is mentioning, keeping the WireGuard GUI closed should resolve the problem

**Comment by QZAiXH** [comment]
Close the GUI, the configuration will not be updated

**Comment by janost** [comment]
Yeah, closing the GUI is not really a proper workaround, because without it netclient will not be able to reconfigure the tunnel, which kind of defeats the purpose of netmaker.

I haven't tested with 0.10.0 yet.

**Comment by ASHwaysama** [comment]
Just tested with 0.10.0, still [MGR] Unable to ingest and encrypt the .conf file

**Comment by afeiszli** [comment]
The Windows agent has been re-factored as of 0.11 and no longer integrates with the WireGuard app. Users should no longer experience this issue.
